README.md

```diff
@@ -33,7 +33,7 @@ This module provides Stackdriver Trace support for Node.js applications. [Stackd
 
 3. Include and start the library *as the very first action in your application*:
 
-        require('@google/cloud-trace').start();
+        require('@google/cloud-trace')().startAgent();
```

Comment:
nit: it would be better if the examples always save the return value from the constructor into a variable to make it obvious that the `startAgent` call is available once an API object has been constructed.